<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetApp</class>
 <widget class="QWidget" name="QtWidgetApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtWidgetApp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PatientHandler" name="patientHandler" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>85</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>85</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ImageHandler" name="imageHandler" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="MeasurementsHandler" name="measurementHandler" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>460</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LogHandler" name="logHandler" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonQuit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>MeasurementsHandler</class>
   <extends>QWidget</extends>
   <header>MeasurementsHandler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PatientHandler</class>
   <extends>QWidget</extends>
   <header>PatientHandler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogHandler</class>
   <extends>QWidget</extends>
   <header>LogHandler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageHandler</class>
   <extends>QWidget</extends>
   <header location="global">ImageHandler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetApp</receiver>
   <slot>RequestQuit()</slot>
  </connection>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetApp</receiver>
   <slot>RequestLoad()</slot>
  </connection>
 </connections>
 <slots>
  <slot>RequestQuit()</slot>
  <slot>RequestLoad()</slot>
 </slots>
</ui>
